tbx/project_styleguide/templates/patterns/molecules/listing/listing--event--no-image.html

```diff
@@ -0,0 +1,3 @@
+{# Component for preview only. #}
```

Comment:
We have used the styleguide to display these kinds of variations everywhere else in the project, so I'd prefer not to add these pattern library only components here, and include them in the styleguide instead - see http://localhost:8000/pattern-library/render-pattern/patterns/styleguide/components/components.html

Reply:
I see! I didn't quite grok that these components & variants would go there too. Thanks! I've moved this into the components.html file in [cb47ff0](https://github.com/torchbox/torchbox.com/pull/190/commits/cb47ff0bf786d34caf18f350dbb2463f9132eb84).

<details><summary>Screenshot</summary>

![image](https://github.com/torchbox/torchbox.com/assets/5073278/02051ddd-77ef-47ab-ac56-a4b2ffed3af9)

</details>